tests/e2e/test_rosacli_upgrade.go

```diff
@@ -259,4 +261,116 @@ var _ = Describe("Cluster Upgrade testing",
 			Expect(output.String()).To(ContainSubstring("are compatible with upgrade"))
 			Expect(output.String()).To(ContainSubstring("Upgrade successfully scheduled for cluster"))
 		})
+
+		It("to upgrade roles/operator-roles and cluster - [id:57444]", labels.Critical, labels.Runtime.Upgrade, func() {
```

Comment:
@jameszwang I believe Yu Wang have a case to cover the y stream upgrade. Can you reach to him to see if there is there any necessary to merge the two cases?

Reply:
@xueli181114  By reading Yu wang's test case(OCP-73731) and script, I think my  two cases are necessary. Yu wang's case is detach/delete policies and re-create them, but my case is upgrading the existed policies. So I don't think his case overlaps with mine.